lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -23,13 +25,17 @@ function multiplyFractions(node) {
     return Node.Status.noChange(node);
   }
   const atLeastOneFraction = node.args.some(
-    arg => Node.Type.isOperator(arg, '/'));
-  const hasPolynomialTerms = node.args.some(
-    arg => Node.PolynomialTerm.isPolynomialTerm(arg));
-  if (!atLeastOneFraction || hasPolynomialTerms) {
+      arg => Node.Type.isOperator(arg, '/'));
+
+  // If term is in the form of <constant fraction> <polynomial term>
+  // do not multiply in order to keep pedagogical style
+  // e.g 1/2 * x^2 -> 1/2 x^2 and not 1x^2 / 2
+  const isLinearFraction = Node.PolynomialTerm.isPolynomialTerm(node.args[1]);
+  const isConstantFraction = Node.Type.isConstantFraction(node.args[0]);
```

Comment:
what are you assuming about `node.args[0]` and `node.args[1]`? I think that `node.args` could be any length and not just 2 items long, right? you should probably just do a check for that too.